test_helpers/tcs/tcs.go

```diff
@@ -0,0 +1,98 @@
+package tcs
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/tarantool/go-tarantool/v2"
+	"github.com/tarantool/go-tarantool/v2/test_helpers"
+)
+
+type Tcs struct {
+	// conn Connection interface for interacting with Tarantool.
+	doer tarantool.Doer
+}
+
+// GetResponse is a response of a get request.
+type GetResponse struct {
+	Data []struct {
+		Path        string
+		Value       string
+		ModRevision int64 `mapstructure:"mod_revision"`
+	}
+}
+
+// StartTcs starts a Tarantool centralized configuration storage.
+// Returns a Tcs instance and a cleanup function.
+func StartTcs(opts test_helpers.StartOpts) (Tcs, func()) {
+	if less, err := test_helpers.IsTarantoolEEVersionLess(3, 3, 0); less || err != nil {
+		log.Fatal("Tarantool EE 3.3+ required", err)
+	}
+	need_clean := opts.WorkDir == ""
+
+	inst, err := test_helpers.StartTarantool(opts)
+	if err != nil {
+		log.Fatalf("Failed to start Tarantool config instance: %v", err)
+	}
+	conn, err := tarantool.Connect(context.Background(), inst.Dialer, tarantool.Opts{})
+	if err != nil {
+		log.Fatal("Can't connect to Tarantool config instance")
+		return Tcs{}, nil
+	}
+
+	return Tcs{conn}, func() {
+		conn.Close()
+		if need_clean {
+			test_helpers.StopTarantoolWithCleanup(inst)
+		} else {
+			test_helpers.StopTarantool(inst)
+		}
+	}
+
+}
+
+// Put implements "config.storage.put" method.
+func (t *Tcs) Put(ctx context.Context, path string, value string) error {
+	req := tarantool.NewCallRequest("config.storage.put").
+		Args([]any{path, value}).
+		Context(ctx)
+	if _, err := t.doer.Do(req).GetResponse(); err != nil {
+		return fmt.Errorf("failed save data to tarantool: %w", err)
+	}
+	return nil
+}
+
+// delete implements "config.storage.delete" method.
+func (t *Tcs) Delete(ctx context.Context, path string) error {
+	req := tarantool.NewCallRequest("config.storage.delete").
+		Args([]any{path}).
+		Context(ctx)
+	if _, err := t.doer.Do(req).GetResponse(); err != nil {
+		return fmt.Errorf("failed delete data from tarantool: %w", err)
+	}
+	return nil
+}
+
+// get implements "config.storage.get" method.
+func (t *Tcs) Get(ctx context.Context, path string) (GetResponse, error) {
```

Comment:
I would expect symmetry with put here.

```suggestion
func (t *Tcs) Get(ctx context.Context, path string) (string, error) {
```

Reply:
Done